packages/orchestration/src/utils/address.js

```diff
@@ -146,6 +146,17 @@ export const parseAccountId = accountId => {
 };
 harden(parseAccountId);
 
+/**
+ * Parse an account ID into a {@link CaipChainId} following CAIP-10 standards.
+ *
+ * @param {AccountId} accountId CAIP-10 account ID
+ * @returns {CaipChainId}
+ */
+export const parseCaipChainId = accountId => {
```

Comment:
"parse" suggests it's returning a structure, but this is just picking the CAIP-2 out of the CAIP-10.

please drop "parse" from the name. consider `chainOfAccountId`. Or `chainOfAccount` if you widen the param to `AccountIdArg`, which would be similarly simple with `parseAccountIdArg`

Reply:
Good feedback, went with `chainOfAccount`